tensorflow/python/ops/array_ops.py

```diff
@@ -28,6 +28,7 @@
 @@saturate_cast
 @@broadcast_dynamic_shape
 @@broadcast_static_shape
+@@broadcast_to
```

Comment:
Let's add this to contrib for one release before we commit to supporting this API forever. To do so, remove this @@ annotation here, revert the change to tensorflow.pbtxt, and add visibility: HIDDEN to the apidef.pbtxt proto. Then, import this symbol (broadcast_to) from https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/framework/__init__.py so it gets exposed as tf.contrib.framework.broadcast_to

Reply:
Can't you now simply do tf_export('tf.contrib.framework.broadcast_to')?

@annarev?

Reply:
@alextp @martinwicke I add `tf_export('contrib.framework.broadcast_to')(broadcast_to)` and that seems to do the work. Although I also need to update `_allowed_symbols` in `tensorflow/contrib/framework/__init__.py` so that the symbol could be exposed.

Reply:
Thanks @annarev. I updated the PR so that it is exposed from `tensorflow/contrib/framework/__init__.py`.